MAUI/DataGrid/grid-events.md

```diff
@@ -0,0 +1,201 @@
+
+This event will be triggered while double tapping a cell in the DataGrid. This event has the [DataGridCellDoubleTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCellDoubleTappedEventArgs.html) as arguments. 
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   CellDoubleTapped="dataGrid_CellDoubleTapped"
+                   ItemsSource="{Binding OrderInfoCollection}" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void dataGrid_CellDoubleTapped(object sender, DataGridCellDoubleTappedEventArgs e)
+{
+    var rowIndex = e.RowColumnIndex.RowIndex;
+    var rowData = e.RowData;
+    var columnIndex = e.RowColumnIndex.ColumnIndex;
+    var column = e.Column;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### CellLongPress event
+
+This event will be triggered while long pressing a cell in the DataGrid. This event has the [DataGridCellLongPressEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCellLongPressEventArgs.html) as arguments. 
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   CellLongPress="dataGrid_CellLongPress"
+                   ItemsSource="{Binding OrderInfoCollection}" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void dataGrid_CellLongPress(object sender,DataGridCellLongPressEventArgs e)
+{
+    var rowIndex = e.RowColumnIndex.RowIndex;
+    var rowData = e.RowData;
+    var columnIndex = e.RowColumnIndex.ColumnIndex;
+    var column = e.Column;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### CellRightTapped event
+This event will be triggered when a right-click mouse gesture is recognized on a cell. This event has the [DataGridCellRightTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCellRightTappedEventArgs.html) as arguments.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   CellRightTapped="SfDataGrid_CellRightTapped"
+                   ItemsSource="{Binding OrderInfoCollection}" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void SfDataGrid_CellRightTapped(object sender, Syncfusion.Maui.DataGrid.DataGridCellRightTappedEventArgs e)
+{
+    var rowIndex = e.RowColumnIndex.RowIndex;
+    var rowData = e.RowData;
+    var columnIndex = e.RowColumnIndex.ColumnIndex;
+    var column = e.Column;
+    var pointerDeviceType = e.PointerDeviceType;
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> The `CellRightTapped` event is only applicable for Windows and macOS.
+
+## Cell Pointer Events
+
+The `Datagrid` provides the following Events to handle mouse pointer interactions to the cells.
+
+* [CellEntered]() : Called when the mouse pointer enters the cell.
+* [CellHovered]() : Called when the mouse pointer is hovered over the cell.
```

Comment:
Called when the mouse pointer hovers over the cell.

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/cccf8b67-4edf-47cf-b1a0-c435543023a7)
Had updated as discussed.